dace/transformation/dataflow/redundant_array.py

```diff
@@ -97,4 +97,107 @@ def gnode(nname):
             RedundantArray._arrays_removed += 1
 
 
+class RedundantArrayInverse(pm.Transformation):
+    """ Implements the redundant array removal transformation, applied
+        when a transient array is copied from and to (from another array),
+        but never used anywhere else. This transformation remove the second
+        array. """
+
+    _arrays_removed = 0
+    _in_array = nodes.AccessNode("_")
+    _out_array = nodes.AccessNode("_")
+
+    @staticmethod
+    def expressions():
+        return [
+            nxutil.node_path_graph(RedundantArray._in_array,
+                                   RedundantArray._out_array)
+        ]
+
+    @staticmethod
+    def can_be_applied(graph, candidate, expr_index, sdfg, strict=False):
+        in_array = graph.nodes()[candidate[RedundantArray._in_array]]
+        out_array = graph.nodes()[candidate[RedundantArray._out_array]]
```

Comment:
Static variables must use the correct class here.

Reply:
Fixed